New Last Batch Policy changes

swetha097 opened the pull request:
- Modify sharding similar to DALI
- A new vector to store the file_names of the complete dataset.
- Handle the various policies , pad_last_batch, stick_to_shard and shard_size using curr_file_idx and read_counter .
- Pass the LBP and pad_last_batch_repeated to the loaders instead of the context creation.

Commit:
NWC

Commit:
Last Batch Policy Changes

swetha097 committed:
Make the last_batch_padded_size as a virtual instead of pure virtual function in current PR - will be extending the support to other readers in the upcoming PR

swetha097 committed:
Remove the last batch policy changes

swetha097 committed:
Merge branch 'swbs/audio/pr9' into swbs/audio/pr10

swetha097 committed:
NWC

swetha097 committed:
Merge branch 'swbs/audio/pr9' into swbs/audio/pr10

swetha097 committed:
Fix minor issues

swetha097 committed:
Fix multiple bugs while testing

swetha097 committed:
Remove a debug print statement

swetha097 committed:
Revert the CopyRight changes in image and video loader files

swetha097 committed:
Minor Fix

swetha097 committed:
Formatting change in file_source_reader.cpp

swetha097 committed:
Resolve PR comments

swetha097 committed:
Minor changes

swetha097 committed:
Fix the Outputs

swetha097 committed:
Merge branch 'swbs/audio/pr10' of https://github.com/swetha097/rocAL into swbs/audio/pr10

swetha097 committed:
Fix NSR

swetha097 committed:
Fix copy_data with max_x1 and max_y1

swetha097 committed:
Merge remote-tracking branch 'swe_fork/swbs/audio/pr7' into swbs/audio/pr10

fiona-gladwin committed:
Minor fix

fiona-gladwin committed:
Merge branch 'swbs/audio/pr7' into swbs/audio/pr10

fiona-gladwin committed:
Fix copy_data 2D

fiona-gladwin committed:
Fix C++ audio unit test for 2D data

swetha097 committed:
Remove the parse_config

SundarRajan28 committed:
Merge branch 'swbs/audio/pr7' into swbs/audio/pr10

swetha097 committed:
Fix error - Too many open files

swetha097 committed:
Merge branch 'swbs/audio/pr10' of https://github.com/swetha097/rocAL into swbs/audio/pr10

fiona-gladwin committed:
Fix slice fill values copy issue

fiona-gladwin committed:
Merge branch 'swbs/audio/pr10' of https://github.com/swetha097/rocAL into swbs/audio/pr10

fiona-gladwin committed:
Fix file list path in API dataloader

fiona-gladwin committed:
Merge branch 'swbs/audio/pr7' of https://github.com/swetha097/rocAL into swbs/audio/pr10

fiona-gladwin committed:
Fix build errors

swetha097 committed:
Fix Copy_Data_2d_ROI

swetha097 committed:
Merge branch 'swbs/audio/pr10' of https://github.com/swetha097/rocAL into swbs/audio/pr10

Commit:
Fix shard_size and audio source evaluation

Commit:
Changes in file_source_reader - to minimize the I/O operations

Commit:
Changes in the variable name

Commit:
Changes in the variable names of the audio source evalution

swetha097 committed:
Use set instead of vector

swetha097 committed:
Minor bug fixes

fiona-gladwin committed:
Minor fixes

Remove changes to update the
filenames vector incase of padding
Fix the pipeline when pad is off and use idx in a continuous manner

fiona-gladwin committed:
Fix drop policy without padding

To skip and batch and start with the previously padded idx in last batch

fiona-gladwin committed:
Fix pytorch iterator - PARTIAL policy

fiona-gladwin committed:
Merge branch 'swbs/audio/pr10' of https://github.com/swetha097/rocAL into lbp_fix_pr10

fiona-gladwin committed:
Fix PARTIAL

Fix issue with batch size greater than dataset size

swetha097 committed:
Reduce overall time for audio source evalution

swetha097 committed:
Fix shard_size and stick to shard issue seen with convergence

fiona-gladwin committed:
Merge branch 'swbs/audio/pr7' of https://github.com/swetha097/rocAL into swbs/audio/pr10

swetha097 committed:
Merge branch 'swbs/audio/pr10_training' into swbs/audio/pr10

swetha097 committed:
Fix shard_size

swetha097 committed:
Changes in pipeline.py and decoders.py

swetha097 committed:
Merge remote-tracking branch 'origin/swbs/audio/pr10_training' into swbs/audio/pr10

swetha097 committed:
Remove print statement

swetha097 committed:
Merge branch 'swbs/audio/pr10' into lbp_fix_pr10

swetha097 committed:
Fix the count_items

swetha097 committed:
Make Sharding similar to DALI

swetha097 committed:
Fix issues with DROP policy by introducing a new vector for padding

swetha097 committed:
Minor fixes

swetha097 committed:
Comment out print statements

swetha097 committed:
Add changes for shard_size LBP testing

swetha097 committed:
Fix DROP Policy with shard_size > 0

swetha097 committed:
Fix Stick_to_Shard=False

swetha097 committed:
Fix PARTIAL policy and code clean up

swetha097 committed:
fix last_batch_padded size when shard_size > 0

swetha097 committed:
Fix Drop policy - we skip the dropped batch in the next epoch

swetha097 committed:
Fix single shard outputs

swetha097 committed:
Remove the commented code and fix the padding code in open()

swetha097 committed:
Remove div by num_shards in decoders.py

swetha097 committed:
Code clean up and formatting

swetha097 committed:
Minor code clean up

swetha097 committed:
code clean up in pytorch.py

fiona-gladwin committed:
Add layout changes for spectrogram

fiona-gladwin committed:
Pass layouts for MelFilterBank

fiona-gladwin committed:
Fix ToDecibels

Pass layouts for ToDecibels

fiona-gladwin committed:
Fix Normalize

fiona-gladwin committed:
Fix build issue

fiona-gladwin committed:
Merge branch 'swbs/audio/pr5_layout' of https://github.com/swetha097/rocAL into swbs/audio/pr7_layout

fiona-gladwin committed:
Fix python unit test

swetha097 committed:
Merge remote-tracking branch 'swe_fork/swbs/audio/pr7_layout' into swbs/lbp_fixes_pr10

swetha097 committed:
Pass LBP to decoders instead of the Pipeline creation

Commit:
Merge branch 'swbs/lbp_fixes_pr10_pass_readers' into swbs/lbp_fixes_pr10

swetha097 committed:
Update pipeline.py - Remove commented code

swetha097 committed:
Update pipeline.py - Remove commented out code

shobana-mcw reviewed (commented):
Please address the review comments. I could see some changes not pertaining to LBP in this PR. Remove those and add in other PR to reduce the number of file changes. 
Yet to review python changes.

swetha097 committed:
Resolve PR comments

SundarRajan28 reviewed (changes requested):
Please run vscode formatter for cpp files and autopep8 formatter for python files.

swetha097 committed:
Merge branch 'swbs/audio/pr7' into swbs/lbp_fixes_pr10

swetha097 committed:
Fix downmix failing case and resolve the issue with merge

swetha097 committed:
Fix issue with file_source_reader.cpp when file_list is not used

swetha097 committed:
Resolve PR comments - Sundar

swetha097 committed:
Fix file_source_reader.cpp

SundarRajan98 committed:
Fix shuffle issues

shobana-mcw reviewed (commented):
Address the review comments.

sbavasab committed:
Adding comments to all if conditions

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10' of https://github.com/swetha097/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
Merge remote-tracking branch 'swe_fork/swbs/audio/pr7' into swbs/lbp_fixes_pr10

swetha097 committed:
Fix a minor warning in file source reader

swetha097 committed:
LBP comments resolution

SundarRajan28 committed:
Merge remote-tracking branch 'upstream/develop' into swbs/lbp_fixes_pr10

SundarRajan28 committed:
Merge branch 'swbs/audio/pr7' into swbs/lbp_fixes_pr10

SundarRajan28 committed:
Formatting changes

swetha097 committed:
Resolving Final Set of PR comments

swetha097 committed:
Combine with OR condition

swetha097 committed:
Remove the pad_last_batch_repeated print statement from decoders.py

SundarRajan98 committed:
Audio PR - Augmentation support [ Mel-Filter Bank and Normalize ] (#130)

* Name change from sample to data

* Change from decoded_data_info to DecodedDataInfo

* Revert "Change the dims[0] and dims[1] positioning for Spectrogram"

This reverts commit d791b9ac77d662f1528088165d2cfdba721a2b4f.

* Remove audio_decoder_factory.cpp file

* Minor change

* Change variable name

* Change copyright year and remove debug statements

* Add Spectrogram Case in unit tests

* Add spectrogram case in python unit tests

* Update the struct variable name in audio files

* Fixing issues with downmix node output

* Adding ROI updation in downmix node

* Adding downmix test case for python unit tests

* Adding downmix and to_decibels test case in C++ tests

* Minor changes

* Change ROCAL_DATA_PATH to exclude rocal_data

* Update ROCAL_DATA_PATH to exclude rocal_data

* Use Pascal case for function names in audio decoder

* Adding mel filter bank augmentation support in rocAL

* Add resample test case and fixed issues

* NWC - NSR + Slice

* Minor changes

* Minor changes

* Adding latest changes to resample and distribution nodes

* Add test suite changes for NSR

* Fixing ROI handle errors with distribution nodes

* Resolve issue with Slice Node

* Add audio path for downmix test case

* Add NSR and Slice in python unit tests

* Resolving review comments

* Add resample and operator overloading test cases in test suite

* Resolving review comments

* Minor change

* Resolving review comments

* Fix review comments

* Modify cmake to have SNDFILE in all capital

* Fix Slice

* Add NSR and slice in audio test suite

* Fix NSR and Slice output issue

* Resolving review comments

* Adding set_seed in master_graph ctor

* Change Name NonSilentRegion to NonSilentRegionDetection

* fix output mismatch for Tensor add Tensor

* Remove nD tensor layouts

* Minor changes

* Add struct for audio info in AudioReadAndDecode

* Fix merge conflict

* Clean up - node_slice.cpp

* Resolve PR comments - 1

* Remove unused variables, add comments, format merge commit

* Resolve internal PR comments

* Changes in node_slice.cpp

* Change Enum fom UINT TO INT type

* Minor change

* Fornat

* Fix pybind issue for normalize

* Changes in node_slice.cpp to move the fill_values to create_node from update_node

* Add _vx to scalar variables in the nsr

* Moving normalize to effects augmentations folder

* Renaming crop_image_info to CropImageInfo

* Remove - actual_host_buffers - Unused

* Rename TimingDBG to TimingDbg

* Move the instances of DecodedDataInfo to its base class LoaderModule

* Fix a WRN msg in master_graph.cpp

* Remove a dangling comment

* Rename _circ_data_info to _circ_buff_data_info

* Adding changes to normalize node

* Add Glob to CMakeLists.txt

* Rename SndFileDecoder to GenericAudioDecoder

* Fix build issues

* Minor change

* Update python API README.md for audio unit test

* Update audio unit test README

* Add debug and changes to normalize node and test case

* Minor fixes for MFB

* Add Normalize and MFB test cases in C++ tests

* Revert "Merge remote-tracking branch 'swe_fork/swbs/audio/pr7' into swbs/audio/pr9"

This reverts commit e480f982957e644d0b452dc372ecfcf5352e899b, reversing
changes made to e5e9ad2e07fbfd4625c08d1e4a7663776051c23d.

* Fix cpp unit test

* Minor Change

* Fix C++ audio unit test for 2D data

* Minor changes

* Minor fix in verify output

* Change in API docs

* Adding missed param in python unit tests

* Revert "Add Glob to CMakeLists.txt"

This reverts commit 47263d94393e9614a5cebcab6c11c804ac7152f8.

* Fix include headers for Audio files

* Fix copy data 2D

* Minor changes

* Pass decoded data info to load routine instead of separate vectors

* Update CHANGELOG.md

* Update CHANGELOG.md

* Change swap_handle_time variable name in loader

* Update the changelog.md

* Update ChangeLog.md

* Update ChangeLog.md

* Update CHANGELOG.md

* Formatting changes

Add comments

* Update doxygen comments

* Move file source reader from readers/image to readers folder

* Update README and add doxygen description

* Update CMakeLists and README for audio test

* Update README for audio test

* Minor fix

* Fix merge from PR 2

* Minor changes shard_count argument name

* Rename set and get functions of data_info to decoded_data_info

* Revert empty line removed in CMakeLists.txt

* Removed prefix original for audio vectors

* Resolve PR comments

* Add @params to all args in pytorch.py

* Fix build issue

* Minor changes in unit test

* Minor changes

* Change ROCAL instaces to rocAL in pytorch.py

* Resolve the PR comments

* Minor changes in decoders.py - Modify the comment for shard_size

* Minor changes

* Address the PR comments

* Address Review comments

* Introduce Audio layouts

* Add layout changes for spectrogram

* Fix the unit tests - c++ & python

* Minor fix

* Adding changes for spec layout changes

* Adding changes to MFB and normalize nodes

* Update node_slice.cpp

* Update node_slice.h

* Fix merge conflicts

* Resolving review comments

* Resolving review comments

* Resolving review comments

---------

Co-authored-by: fiona-gladwin <fionagladwin@multicorewareinc.com>
Co-authored-by: Swetha B S <>
Co-authored-by: Swetha B S <swetha@multicorewareinc.com>
Co-authored-by: SundarRajan28 <sundarrajan@multicorewareinc.com>
Co-authored-by: Fiona-MCW <70996026+fiona-gladwin@users.noreply.github.com>
Co-authored-by: swetha097 <59434434+swetha097@users.noreply.github.com>
Co-authored-by: Lakshmi Kumar <lakshmi.kumar@amd.com>
Co-authored-by: Kiriti Gowda <kiritigowda@gmail.com>

fiona-gladwin committed:
Merge branch 'develop' of https://github.com/ROCm/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
Add shard_size and stick_to_shard variables in args

SundarRajan28 committed:
Commenting test cases in unit tests (#180)

Co-authored-by: SundarRajan98 <svaithiy@amd.com>
Co-authored-by: Lakshmi Kumar <lakshmi.kumar@amd.com>

swetha097 committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

swetha097 committed:
Minor spelling fix

SundarRajan28 committed:
Updating TF pets training example (#179)

* Add tf_pets training example

* Adding dockerfile and readme changes

* Resolving review comments

* Resolving review comments

---------

Co-authored-by: SundarRajan98 <svaithiy@amd.com>

kiritigowda committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

swetha097 committed:
Make changes to insert the padded data in the file_names vector

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10' of https://github.com/swetha097/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
Support to pass the variables fo lbp as struct

dependabot[bot] committed:
Docs - Bump rocm-docs-core[api_reference] from 1.5.0 to 1.5.1 in /docs/sphinx (#183)

Bumps [rocm-docs-core[api_reference]](https://github.com/ROCm/rocm-docs-core) from 1.5.0 to 1.5.1.
- [Release notes](https://github.com/ROCm/rocm-docs-core/releases)
- [Changelog](https://github.com/ROCm/rocm-docs-core/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/ROCm/rocm-docs-core/compare/v1.5.0...v1.5.1)

---
updated-dependencies:
- dependency-name: rocm-docs-core[api_reference]
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

kiritigowda committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

swetha097 committed:
Fix segmentation fault

SundarRajan28 committed:
Merge branch 'ROCm:develop' into swbs/lbp_fixes_pr10_pr_comments

amd-jnovotny committed:
Docs - Fix a few erroneous links to the examples directory (#187)

kiritigowda committed:
Setup - SLES 15 SP5 Updates (#188)

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10_pr_comments' into swbs/lbp_fixes_pr10

swetha097 committed:
Resolve PR comments

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10' of https://github.com/swetha097/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
Resolve PR comments

swetha097 committed:
Resolve PR comments

swetha097 committed:
Use PreComputed start and end indices

swetha097 committed:
Use precomputed shard_idx start and end in initialize

kiritigowda committed:
RHEL - fix FFMPEG (#190)

swetha097 committed:
Audio readme update (#186)

* Audio Doc addition

* Update the slice

---------

Co-authored-by: Lakshmi Kumar <lakshmi.kumar@amd.com>

kiritigowda committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

dependabot[bot] committed:
Bump rocm-docs-core[api_reference] from 1.5.1 to 1.6.1 in /docs/sphinx (#185)

Bumps [rocm-docs-core[api_reference]](https://github.com/ROCm/rocm-docs-core) from 1.5.1 to 1.6.1.
- [Release notes](https://github.com/ROCm/rocm-docs-core/releases)
- [Changelog](https://github.com/ROCm/rocm-docs-core/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/ROCm/rocm-docs-core/compare/v1.5.1...v1.6.1)

---
updated-dependencies:
- dependency-name: rocm-docs-core[api_reference]
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Kiriti Gowda <kiritigowda@gmail.com>

dependabot[bot] committed:
Docs - Bump rocm-docs-core[api_reference] from 1.6.1 to 1.6.2 in /docs/sphinx (#198)

Bumps [rocm-docs-core[api_reference]](https://github.com/ROCm/rocm-docs-core) from 1.6.1 to 1.6.2.
- [Release notes](https://github.com/ROCm/rocm-docs-core/releases)
- [Changelog](https://github.com/ROCm/rocm-docs-core/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/ROCm/rocm-docs-core/compare/v1.6.1...v1.6.2)

---
updated-dependencies:
- dependency-name: rocm-docs-core[api_reference]
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Docs - Bump rocm-docs-core[api_reference] from 1.6.2 to 1.7.0 in /docs/sphinx (#200)

Bumps [rocm-docs-core[api_reference]](https://github.com/ROCm/rocm-docs-core) from 1.6.2 to 1.7.0.
- [Release notes](https://github.com/ROCm/rocm-docs-core/releases)
- [Changelog](https://github.com/ROCm/rocm-docs-core/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/ROCm/rocm-docs-core/compare/v1.6.2...v1.7.0)

---
updated-dependencies:
- dependency-name: rocm-docs-core[api_reference]
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

kiritigowda committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

swetha097 committed:
Merge remote-tracking branch 'origin/swbs/lbp_fixes_pr10_pr_comments' into swbs/lbp_fixes_pr10

dependabot[bot] committed:
Docs - Bump rocm-docs-core[api_reference] from 1.7.0 to 1.7.1 in /docs/sphinx (#202)

Bumps [rocm-docs-core[api_reference]](https://github.com/ROCm/rocm-docs-core) from 1.7.0 to 1.7.1.
- [Release notes](https://github.com/ROCm/rocm-docs-core/releases)
- [Changelog](https://github.com/ROCm/rocm-docs-core/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/ROCm/rocm-docs-core/compare/v1.7.0...v1.7.1)

---
updated-dependencies:
- dependency-name: rocm-docs-core[api_reference]
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

samjwu committed:
CI - Run python as sudo (#201)

For elevated perm in sles pipeline

Co-authored-by: Kiriti Gowda <kiritigowda@gmail.com>

dependabot[bot] committed:
Docs - Bump rocm-docs-core[api_reference] from 1.7.1 to 1.7.2 in /docs/sphinx (#205)

Bumps [rocm-docs-core[api_reference]](https://github.com/ROCm/rocm-docs-core) from 1.7.1 to 1.7.2.
- [Release notes](https://github.com/ROCm/rocm-docs-core/releases)
- [Changelog](https://github.com/ROCm/rocm-docs-core/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/ROCm/rocm-docs-core/compare/v1.7.1...v1.7.2)

---
updated-dependencies:
- dependency-name: rocm-docs-core[api_reference]
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

kiritigowda committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

swetha097 committed:
Initialize the Sharding info using ShardingInfo()

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10' of https://github.com/swetha097/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
convert the signed to int32_t type

swetha097 committed:
temp commit for struct changes

SundarRajan28 committed:
Spectrogram example notebook (#181)

* Adding spectrogram notebook

* Minor change

* Resolving review comments

---------

Co-authored-by: Kiriti Gowda <kiritigowda@gmail.com>

Commit:
Fix the struct changes - All the test cases passing

Commit:
Remove any print statements

Commit:
Add support to Pass the decode size policy from the user

Commit:
Add support to Pass the decode size policy from the user

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10' of https://github.com/swetha097/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
Rename RocalShardingInfo for ShardingInfo and vice-versa

swetha097 committed:
xywh roi copy

swetha097 committed:
Fix decoders.py for image decoders

swetha097 committed:
Make stick_to_shard True by default

fgladwin committed:
Merge branch 'develop' of https://github.com/ROCmSoftwarePlatform/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
Minor changes to the copy_data function

fgladwin committed:
Rename to x_offset and y_offset in copy_data

fgladwin committed:
Minor changes - remove unused variables

fgladwin committed:
Minor change - Variable names

fgladwin committed:
Update Doxygen comments and comments of API

fgladwin committed:
Merge branch 'swbs/lbp_fixes_pr10' of https://github.com/swetha097/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
Audio decoder Jupyter NoteBook (#192)

* Audio Decoder Jupyter NB

(cherry picked from commit afe0370b179b841e885739d5687fc73f59431236)

* Remove the outputs from audio decoder jupyter NB

* Modifying the comments in ipynb for audio decoder

* Modify the .ipynb and address the comments.

* Resolve the comments

* Merge

* Beautify the Jupyter NoteBook

---------

Co-authored-by: root <root@smc300x-ccs-aus-GPUF2D0.cs-aus.dcgpu>
Co-authored-by: Kiriti Gowda <kiritigowda@gmail.com>
Co-authored-by: Swetha B S <swetha@multicorewarein.com>
Co-authored-by: Swetha B S <swetha@multicorewaresinc.com>
Co-authored-by: Lakshmi Kumar <lakshmi.kumar@amd.com>

LakshmiKumar23 committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

SundarRajan28 committed:
pipeline creation -  fix for device_id (#207)

Co-authored-by: Kiriti Gowda <kiritigowda@gmail.com>
Co-authored-by: Lakshmi Kumar <lakshmi.kumar@amd.com>

kiritigowda committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

swetha097 committed:
pybind  - Fix external source reader (#199)

* Fix external source reader pybind call

* Add gpu backend support in external_source_reader.py

* Add last batch info for all the video readers

* Move import ciupy to LINE 44

* Fix the image extensions

---------

Co-authored-by: Kiriti Gowda <kiritigowda@gmail.com>
Co-authored-by: Lakshmi Kumar <lakshmi.kumar@amd.com>

SundarRajan28 committed:
pytorch training example - Update  (#206)

* Adding calculate_ips mode to training example

* Add assert statement

---------

Co-authored-by: Lakshmi Kumar <lakshmi.kumar@amd.com>
Co-authored-by: Kiriti Gowda <kiritigowda@gmail.com>

Commit:
Make the rocalShardingInfo as the last param for Audio loaders

Commit:
Remove unused variables and functions in file_source_reader.cpp & .h files

Commit:
Remove the doctring explanation for unused params

Commit:
Change the explanation according to the newly introduced structure

LakshmiKumar23 committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10
